.pre-commit-config.yaml

```diff
@@ -0,0 +1,31 @@
+
+repos:
+- repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v3.4.0
+  hooks:
+  - id: check-added-large-files
+  - id: check-case-conflict
+  - id: check-toml
+  - id: check-yaml
+  - id: debug-statements
+  - id: end-of-file-fixer
+    exclude: WHEEL
+    exclude_types: [json]
+  - id: forbid-new-submodules
```

Comment:
I don't see this in the pre-commit docs. I can guess what it does, but could be wrong. What effect will this have on our workflow?

Reply:
It "prevents addition of new git submodules", should have no effect other than preventing a foot-gun.

Reply:
Here's the full list of hooks docs: https://pre-commit.com/hooks.html

Reply:
Ah, I was looking at the wrong docs. Thanks.